src/header/css/header-inline-title.pcss

```diff
@@ -0,0 +1,284 @@
+@import "../../shared/css/shared.pcss";
+
+/* Global Header Inline Title 
+/* ========================================================================== */
+
+.esri-header-inline-title {
+	@apply --unset-a;
+	align-items: center;
+	display: flex;
+	flex: 0 0 auto;
+	height: 100%;
+	text-decoration: none;
+	color: var(--mono-35);
+
+	&.-fit-burger{
+		position: relative;
+	}
+
+	&:empty {
+		display: none;
+	}
+
+	@media (--width-s) {
+		margin-inline-end: auto;
+		padding-inline: 12px;
+		padding-inline-start: 6px;
+	}
+
+	@media (--width-m) {
+		margin-inline-end: auto;
+		padding-inline-end: 12px;
+		padding-inline-start: 6px;
+	}
+
+	@media (--width-l) {
+		padding-inline-start: 8px;
+		padding-inline-end: 12px;
+	}
+}
+
+/* Global Header Title: Text
+/* ========================================================================== */
+
+.esri-header-inline-title-text {
+	font-size: 18px;
+	font-weight: 400;
+	color: var(--mono-20);
+	overflow: hidden;
+	display: inline-block;
+	text-overflow: ellipsis;
+	white-space: nowrap;
+
+	@media (--width-s) {
+		max-width: 70vw !important;
+	}
+
+	@media (--width-m) {
+		max-width: 61vw !important;
```

Comment:
does this work without the `!important`?

Reply:
Unfortunately, yes 😞 . I'm currently letting the user modify the `max-width` with the maxViewWidth property. However, that property becomes irrelevant once the menu links disappear at medium/small sizes. So I'm overriding it to make sure the title scales correctly at those sizes.